docs/examples/notebooks/visualization/surface_code_demo.ipynb

```diff
@@ -0,0 +1,587 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8112ffee",
+   "metadata": {},
+   "source": [
+    "# Interactive Surface Codes with `deltakit` and `crumpy`\n",
+    "\n",
+    "In this notebook, we show how `deltakit` can be used to generate surface code memory experiments and various ways in which visualization packages like [`crumpy`](https://pypi.org/project/crumpy/) and [`plotly`](https://pypi.org/project/plotly/) can be used to create interactive interfaces for surface code exploration. We show examples of generating planar codes with `deltakit.explorer.codes`, how to interactively visualize surface code experiment circuits with `crumpy` and `plotly`, and how to add Pauli error markers to those generated circuits to explore the error-detecting capabilities of surface codes.\n",
+    "\n",
+    "## Required Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6b9a8078-23b0-4062-a951-bd96208e82ad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from deltakit.explorer.codes import RotatedPlanarCode, UnrotatedPlanarCode, css_code_memory_circuit\n",
```

Comment:
This is not specific to this notebook, but I realize that the default line length before a scroll bar gets added is 79 characters.

<img width="738" height="318" alt="image" src="https://github.com/user-attachments/assets/357c973f-796c-4d9b-80d3-026ba5665cde" />

No need to change anything now; just an observation.